italic_block.py

```diff
@@ -10,3 +11,30 @@ def __init__(self, content):
 
     def render(self) -> str:
         return f"<em>{self.content()}</em>"
+
+    @staticmethod
+    def parse(content) -> (int, int, 'ItalicBlock'):
+        """Parse a italic content, and return (begin, end, ItalicBlock) if parseable,
+        or (-1, -1, None) that no such a element
+        """
+        pattern_star = r'(^|[^\\])([*])(.*[^\\])([*])'
```

Comment:
Just FYI, since we won't use the content the the Group 1 `(^|[^\\])`, you can use non-catch group to create this matching group but it won't be catch.

Syntax: rather than using `( )`, use `(?: )`.

If you use non-catch group, then `start = match.start(1)` rather than `start = match.start(2)`

Reply:
Addressed